c2cgeoform/ext/deform_ext.py

```diff
@@ -423,6 +428,10 @@ class RelationCheckBoxListWidget(RelationMultiSelectMixin,
         The property of the model that is used for the ``order_by`` clause of
         the SQL query.
         Default: ``None``.
+    edit_url (optionnal)
+        function
+        in :  request and values
+        return: url
```

Comment:
I would prefer a phrase : a function taking request and value as parameter and returning an url to the correponding resource.